modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SkuResourceImpl.java

```diff
@@ -339,9 +360,18 @@ private Sku _updateSKU(CPInstance cpInstance, Sku sku) throws Exception {
 
 		DateConfig expirationDateConfig = new DateConfig(expirationCalendar);
 
+		String externalReferenceCode = cpInstance.getExternalReferenceCode();
+		String skuExternalReferenceCode = sku.getExternalReferenceCode();
+
+		if ((skuExternalReferenceCode != null) &&
+			!skuExternalReferenceCode.isEmpty()) {
+
+			externalReferenceCode = skuExternalReferenceCode;
+		}
```

Comment:
no this would be run with the patch byERC which is not in the acceptance criteria